pkgs/applications/audio/calf/default.nix

```diff
@@ -16,8 +16,12 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [
-    cairo expat fftwSinglePrec fluidsynth glib gtk2 libjack2 ladspaH
-    libglade lv2
+    expat fftwSinglePrec fluidsynth glib libjack2 ladspaH lv2
+  ];
+
+  configureFlags = [
+    # depends on gtk2
+    "--without-gui"
```

Comment:
Yeah.  If you can make it work with GTK3 (which I think should be possible) then I'm all for it, but otherwise this would break things.

Reply:
It [does not look like GTK 3 is supported](https://github.com/calf-studio-gear/calf/blob/c41d8c0d033d886df3de8d419edfd1c2a942170f/configure.ac#L102-L105) by calf. [Last time](https://github.com/NixOS/nixpkgs/pull/159875#issuecomment-1038384634) I checked, the LV2 plug-in ecosystem was pretty averse to GTK 3 migration because it cannot run alongside GTK 2 so all plug-ins would need to be ported at the same time.

Reply:
easyeffects useses some library part of this in a dependency of a dependency. Maybe there splitting the GUI and lib would already help.